exercises/arrays/lodashCompact/index.ts

```diff
@@ -0,0 +1,8 @@
+type FalsyValues = 0 | null | false;
```

Comment:
Some values are  missed
https://developer.mozilla.org/en-US/docs/Glossary/Falsy#:~:text=A%20falsy%20(sometimes%20written%20falsey,such%20as%20conditionals%20and%20loops.&text=The%20keyword%20false%20.&text=The%20Number%20zero%20(so%2C%20also,etc.%2C%20and%200x0%20).